tests/tools/account.test.ts

```diff
@@ -180,3 +181,17 @@ describe('getAccountCharsetTypes', function () {
     })
   })
 })
+
+describe('digitalEmojiUnifiedHandle', function () {
+  it('0⃣️.bit', () => {
+    expect(digitalEmojiUnifiedHandle('0⃣️.bit')).toEqual('0️⃣.bit')
+  })
+
+  it('1️⃣.bit', () => {
+    expect(digitalEmojiUnifiedHandle('1️⃣.bit')).toEqual('1️⃣.bit')
+  })
+
+  it('2⃣.bit', () => {
```

Comment:
There can be another test, `imac0011️⃣.bit`, to prevent affecting the normal account.